app/services/news_service.py

```
@@ -1,9 +1,367 @@
+            if full_content and len(full_content) > len(article.get("content", "")):
+                # Replace truncated content with full content
+                article["content"] = full_content
+                article["content_source"] = "scraped"
+                logger.info(f"Successfully scraped content from {url}")
+            else:
+                # Keep Original content if scraping fails or content is shorter
+                article["content_source"] = "api"
+                logger.warning(f"Scraping returned insufficient content for {url}")
+        except Exception as e:
+            logger.warning(
+                f"Failed to scrape content from {article.get("url", "")}: {str(e)}"
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: unmatched '(' (news_service, line 336)'

(E0001)